tests/test_edr.py

```diff
@@ -163,6 +163,14 @@ def _assert_progress_range(self, progress, dt, start, stop, step):
             assert ref_line == progress_line
 
 
+def test_edr_to_dict():
```

Comment:
This only really tests that it returns the same as `edr_to_df` so perhaps a more descriptive name like `edr_to_dict_matches_edr_to_df`. I know its a bit verbose but clarity is always good.